00-get-started/01-ingest-kafka-data.md

```diff
@@ -67,38 +67,50 @@ WITH (
+Optionally, for verification, you can create a materialized view to grab all existing data from the source, using the following SQL.
+```sql
+CREATE MATERIALIZED VIEW IF NOT EXISTS verify_website_visits AS
+  SELECT * FROM website_visits_stream;
+```
 
-### Analyze the data
+By running `SELECT * FROM verify_website_visits;`, you should see the outputs as follows.
+```terminal
+         timestamp         | user_id | page_id | action
+---------------------------+---------+---------+--------
+ 2023-06-13 10:05:00+00:00 | user1   | page1   | click
+ 2023-06-13 10:06:00+00:00 | user2   | page2   | scroll
+ 2023-06-13 10:07:00+00:00 | user3   | page1   | view
+ 2023-06-13 10:08:00+00:00 | user4   | page2   | view
+ 2023-06-13 10:09:00+00:00 | user5   | page3   | view
+(5 rows)
+```
 
-By creating a source, RisingWave has been connected to the data stream. However, in order to ingest and process the data from Kafka, we need to create some materialized views. Each of these materialized views contains the results of a query, which are updated as soon as new data comes in.
+To learn more about the `CREATE SOURCE` command, check [`CREATE SOURCE`](https://docs.risingwave.com/docs/current/sql-create-source/) from the offical RisingWave documentation.
 
-The following SQL query creates a materialized view named `visits_stream_mv` based on the source `website_visits_stream`. For each `page_id`, it calculates the number of total visits, the number of unique visitors, and the timestamp when the page was most recently visited.
+To further perform some basic analysis on the data from the created source, check [Section 00-01](../01-query-process-streaming-data/001-ingest-analyze-kafka.md#analyze-the-data).
 
-```sql
-CREATE MATERIALIZED VIEW visits_stream_mv AS
- SELECT page_id,
- COUNT(*) AS total_visits,
- COUNT(DISTINCT user_id) AS unique_visitors,
- MAX(timestamp) AS last_visit_time
- FROM website_visits_stream
- GROUP BY page_id;
-```
+### Create a table
 
-We can query from the materialized view to see the results.
+You can also create a table to connect to the Kafka topic. Compared to creating a source, a table persists the data from the stream by default. In this way, you can still query the table data even after the Kafka environment has been shut down.
 
+The following SQL query creates a table named `website_visits_table`, using the `CREATE TABLE` command.
 ```sql
-SELECT * FROM visits_stream_mv;
+CREATE TABLE IF NOT EXISTS website_visits_table (
+ timestamp timestamptz,
+ user_id VARCHAR,
+ page_id VARCHAR,
+ action VARCHAR
+ )
+WITH (
+ connector='kafka',
+ topic='test',
+ properties.bootstrap.server='localhost:9092',
+ scan.startup.mode='earliest'
+) FORMAT PLAIN ENCODE JSON;
 ```
 
-The results will look like the following. Note that the rows do not necessarily follow this order.
+For verification, run `SELECT * FROM website_visits_table;` and you should see the same outputs in the [`Create a source` part](#create-a-source).
 
```

Comment:
Maybe also add suggestions on when sources/tables are preferred?

Reply:
Yes, I will write a pr to compare them later.